test/testoverlay2.c

```diff
@@ -219,14 +218,35 @@ void loop()
     SDL_Delay(fpsdelay);
 #endif
 
+
+#if 1
```

Comment:
Can you add a symbol for this to switch which codepath you're testing? Or a command line option?